tests not working

tests not running with `npm test`, issue with module resolution. Need to check tsconfig

``` 
FAIL  src/features/account/controller/index.spec.ts
  ● Test suite failed to run

    Cannot find module './index.js' from 'src/features/account/controller/index.spec.ts'

      1 | import httpMocks from 'node-mocks-http';
      2 |
    > 3 | import {
        | ^
      4 |     listAccountsController,
      5 |     getAccountController,
      6 |     createAccountController,

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (src/features/account/controller/index.spec.ts:3:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        1.25 s
Ran all test suites.
```

--- a/server/jest.config.js
+++ b/server/jest.config.js
@@ -9,5 +9,8 @@ export default {
     '!<rootDir>/src/**/*.spec.ts',
     '!<rootDir>/src/**/*.test.ts',
     '!<rootDir>/src/**/*.d.ts'
-  ]
+  ],
+  moduleNameMapper: {
+    '(.+)\\.js': '$1'
+  },
 };

